service/history/fx.go

```diff
@@ -84,8 +91,22 @@ var Module = fx.Options(
 	fx.Provide(HandlerProvider),
 	fx.Provide(ServiceProvider),
 	fx.Invoke(ServiceLifetimeHooks),
+
+	fx.Provide(CallbackExecutorOptionsProvider),
+	callbacks.Module,
 )
 
+func CallbackExecutorOptionsProvider() callbacks.ActiveExecutorOptions {
```

Comment:
I'm debating moving this into the callbacks plugin. This is a dependency of the callback executor.